packages/completer/src/reconciliator.ts

```diff
@@ -17,7 +17,8 @@ import { Completer } from './widget';
 import { Signal } from '@lumino/signaling';
 
 // Shorthand for readability.
-type InlineResult = IInlineCompletionList<CompletionHandler.IInlineItem> | null;
+export type InlineResult =
```

Comment:
Not sure if we need to export this, it is just a shorthand.

Reply:
Typedoc is complaining because it does not resolve shorthand...